test/unit/test_features.py

```diff
@@ -24,6 +24,9 @@ def check_sign_ext(self, module, error):
   def check_bulk_mem(self, module, error):
     self.check_feature(module, error, '--enable-bulk-memory')
 
+  def check_exception_handling(self, module, error):
+    self.check_feature(module, error, '--enable-exception-handling')
```

Comment:
Is this meant to be in preparation for adding a test in the future? I suppose adding a test would be blocked on being able to parse/print a wasm using exception handling.

Reply:
Yes, I added it because now we would be able to at least pass `--enable/disable-exception-handling`, but to use it I need instructions, so it's no much use. Do you want me to remove it here?

Reply:
Seems fine to keep it in.